packages/coinbase/src/index.ts

```diff
@@ -13,12 +13,22 @@ function coinbaseWallet({
         const [chain] = chains
         const { name, icon } = appMetadata || {}
 
-        const { CoinbaseWalletSDK } = await import('@coinbase/wallet-sdk')
+        // according to https://github.com/wagmi-dev/wagmi/issues/383
+        // @coinbase/wallet-sdk export double default fields
+        // so we need to detect it to get the real constructor
+        const { default: CoinbaseWalletSDK } = await import(
+          '@coinbase/wallet-sdk'
+        )
+        const CoinbaseWalletSDKConstructor = (
+          (CoinbaseWalletSDK as any).default
```

Comment:
@Adamj1232 if you check for `'default' in` I think you could forgo all of the casting ya?

Reply:
@duxiaofeng-github ☝🏼

Reply:
Would still get a type error getting trying to access `.default` after checking using `in`